utils/oomparser/oomparser_test.go

```diff
@@ -126,42 +118,203 @@ func TestCheckIfStartOfMessages(t *testing.T) {
+					"204 total pagecache pages",
+					"0 pages in swap cache",
+					"Swap cache stats: add 0, delete 0, find 0/0",
+					"Free swap  = 0kB",
+					"Total swap = 0kB",
+					"445340 pages RAM",
+					"0 pages HighMem/MovableOnly",
+					"10614 pages reserved",
+					"[ pid ]   uid  tgid total_vm      rss nr_ptes swapents oom_score_adj name",
+					"[  273]     0   273     4869       50      13        0             0 upstart-udev-br",
+					"[  293]     0   293    12802      154      28        0         -1000 systemd-udevd",
+					"[  321]     0   321     3819       54      12        0             0 upstart-file-br",
+					"[  326]   102   326     9805      109      24        0             0 dbus-daemon",
+					"[  334]   101   334    63960       94      26        0             0 rsyslogd",
+					"[  343]     0   343    10863      102      26        0             0 systemd-logind",
+					"[  546]     0   546     3815       60      13        0             0 upstart-socket-",
+					"[  710]     0   710     2556      587       8        0             0 dhclient",
+					"[  863]     0   863     3955       48      13        0             0 getty",
+					"[  865]     0   865     3955       50      13        0             0 getty",
+					"[  867]     0   867     3955       51      13        0             0 getty",
+					"[  868]     0   868     3955       51      12        0             0 getty",
+					"[  870]     0   870     3955       49      13        0             0 getty",
+					"[  915]     0   915     5914       61      16        0             0 cron",
+					"[ 1015]     0  1015    10885     1524      25        0             0 manage_addresse",
+					"[ 1028]     0  1028     3955       49      13        0             0 getty",
+					"[ 1033]     0  1033     3197       48      12        0             0 getty",
+					"[ 1264]     0  1264    11031     1635      26        0             0 manage_accounts",
+					"[ 1268]     0  1268    15341      180      33        0         -1000 sshd",
+					"[ 1313]   104  1313     6804      154      17        0             0 ntpd",
+					"[ 1389]     0  1389    25889      255      55        0             0 sshd",
+					"[ 1407]  1020  1407    25889      255      52        0             0 sshd",
+					"[ 1408]  1020  1408     5711      581      17        0             0 bash",
+					"[ 1425]     0  1425    25889      256      53        0             0 sshd",
+					"[ 1443]  1020  1443    25889      257      52        0             0 sshd",
+					"[ 1444]  1020  1444     5711      581      16        0             0 bash",
+					"[ 1476]  1020  1476     1809       25       9        0             0 tail",
+					"[ 1532]  1020  1532   410347   398810     788        0             0 badsysprogram",
+					"Out of memory: Kill process 1532 (badsysprogram) score 919 or sacrifice child",
+					"Killed process 1532 (badsysprogram) total-vm:1641388kB, anon-rss:1595164kB, file-rss:76kB",
+				},
```

Comment:
Add a test case with multiple OOM events (note: I think writeAll needs to run in a goroutine to prevent deadlock)